mesh/mesh.hpp

```diff
@@ -1409,6 +1409,12 @@ class Mesh
    /// should not be deleted by the caller.
    static FiniteElement *GetTransformationFEforElementType(Element::Type);
 
+   /** @brief For the face of a boundary element with the orientation @a o (3D),
```

Comment:
Could you mention that the function handles both internal and external boundaries?